### DISCUSSION TITLE
Open a File from mini.files popup

### DISCUSSION THREAD

**Discussion by augustocdias** [nvim-mini/mini.nvim#1063] [Q&A] [answered]
Is there a way to open a file from the mini.files popup?

I often use it to navigate as well and find myself willing to open a file...

**Comment by echasnovski** [comment, marked as answer]
Yes, try pressing `l` and `L` with default mappings. See [Navigation overview](https://github.com/echasnovski/mini.nvim/blob/f20d8cd3a116ef65f022797de064a044b217ff53/doc/mini-files.txt#L107) and [video demo](https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-files.md#demo).

**Reply by augustocdias** [reply]
Thanks a lot. I completely misinterpreted this behavior.
